src_c/event.c

```diff
@@ -1863,7 +1864,7 @@ MODINIT_DEFINE(event)
             MODINIT_ERROR;
         }
 
-        if (SDL_RegisterEvents(PGE_NUMEVENTS) != PGE_EVENTBEGIN) {
+        if (SDL_RegisterEvents(PGE_NUMEVENTS) != SDL_NUMEVENTS) {
```

Comment:
This second [`SDL_RegisterEvents`](https://wiki.libsdl.org/SDL_RegisterEvents) call should just be removed now, or else it must be called before the above `SDL_RegisterEvents` invocation (because its range now comes first). Also, `PGE_EVENTBEGIN` should still work here (returns first event).

But the easiest fix is to just remove this if statement altogether.

Reply:
Should I change 2xSDL_USERVENT to PGE_USEREVENT in this IS_SDLv2 if statements?

Reply:
No. We're reserving the values in the interval [SDL_USEREVENT, PGE_USEREVENT) as well as in [PGE_USEREVENT, SDL_NUMEVENTS), so this should be enough:

```c
#if IS_SDLv2
    if (!have_registered_events) {
        int numevents = SDL_NUMEVENTS - SDL_USEREVENT;
        Uint32 user_event = SDL_RegisterEvents(numevents);

        if (user_event == (Uint32)-1) {
            PyErr_SetString(pgExc_SDLError, "unable to register user events");
            DECREF_MOD(module);
            MODINIT_ERROR;
        }
        if (user_event != SDL_USEREVENT) {
            PyErr_SetString(PyExc_ImportError,
                            "Unable to create another module instance");
            DECREF_MOD(module);
            MODINIT_ERROR;
        }

        have_registered_events = 1;
    }

    SDL_SetEventFilter(pg_event_filter, NULL);
#endif /* IS_SDLv2 */
```